### PR TITLE
Proposed changes to app

### PR DESCRIPTION
**Pull request by kfbow**
Here are some small folder structure cleanup and proposed css changes.

**Images:**
screen shot 2018-02-20 at 5 09 08 pm

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -98,9 +98,9 @@ if (process.env.NODE_ENV === 'production') {
     }),
     new CopyWebpackPlugin([
       { from: 'manifest.json', to: 'manifest.json' },
-      { from: 'coin.png', to: 'coin.png'},
-      { from: 'coins.html', to: 'coins.html' },
-      { from: 'icons/*.png', to: ''},
+      { from: 'assets/images/coin.png', to: 'coin.png'},
```
Comment (kfbow) [inline webpack.config.js]: @usmanity are you using these assets anywhere?

File: src/App.vue [hunk 2]
```diff
@@ -41,8 +41,6 @@ export default {
 
 <style lang="scss">
 body {
-  width: 300px;
-  height: 470px;
```
Comment (kfbow) [inline src/App.vue]: Moved into `#app` to center the app

File: webpack.config.js (hunk 1 above)
Comment (usmanity) [reply webpack.config.js]: yes, these get used as the extension's icons

**Comment by usmanity** [comment]
how does it look when it's a chrome extension now?

**Comment by kfbow** [comment]
@usmanity good call haha let me check

**Comment by kfbow** [comment]
@usmanity looks good 👍

**Review by usmanity — APPROVED** [review]

**Pull request merged**